src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
First of all that is not the basis of the argument you have been making.  Which is that the file can become corrupted.  Which goes for both equally so if one can not be trusted then neither can the other.

Secondly they are both basically static files.  These certs only get changed if the user changes them.  Just like the CA bundle only gets changed if the package changes it.  In which case they get rewritten.  Otherwise they are static.  Just because it's done through the GUI does not change that or make it any different than a power user adding it manually.

It's not that I don't get it.  It's that the reasoning is inconsistently applied and not logical.

It doesn't matter that this code doesn't touch the CA bundle.  Just like the CA bundle, once it's there it's there and this code doesn't need to touch them again until the user makes a change.  Just like the CA bundle doesn't need to be touched again until the package changes it.